utils.sh

```diff
@@ -44,6 +45,14 @@ function create_cluster() {
 
     cp ${assets_dir}/install-config.yaml{.tmp,}
     $OPENSHIFT_INSTALLER --dir "${assets_dir}" --log-level=debug create cluster
+
+    # Clean up
+    [ -f assets/files/etc/chrony.conf.tmp ] && \
+      mv assets/files/etc/chrony.conf{.tmp,}
+
+    for file in assets/templates/{99_worker-chronyd-custom.yaml,99_master-chronyd-custom.yaml}; do
+      [ -f ${file} ] && rm ${file}
+    done
```

Comment:
These are already cleaned up in `ocp_cleanup.sh` but wouldn't it be easier/better to stop messing with the assets/files completey, and instead modify/copy things into assets/generated (e.g after generate_assets runs, which already cleans that directory, and it's already ignored by git)

Reply:
As `chrony.conf` needs to be modified after the yaml files are templated, I opted of doing it 'manually' instead.
Let me know about this solution.
Thanks @hardys !